actions/crib-purge-environment/README.md

```diff
@@ -1,3 +1,19 @@
-# crib-purge-environment
+# CRIB Purge Environment action
 
-> Action to destroy CRIB epehemeral environment
+## Description
+
+The **`crib-purge-environment`** action is designed to destroy a CRIB ephemeral
+environment. It should be run after the `crib-deployment-environment` action to
+clean up resources. This action depends on the environment setup provided by the
+dependent composite action.
+
+## Inputs
```

Comment:
Wondering if we should just mention the actions.yml for all of the inputs. I worry we will update them there and forget to update this table.